p2p_wallet/Scenes/Main/Swap/Providers/JupiterTokensRepository.swift

```diff
@@ -3,42 +3,58 @@ import Combine
 import SolanaSwift
 import Resolver
 
+/// Tokens data returned by Jupiter
```

Comment:
(Optional) I know that you are in the middle of very intensive tasks/features.
Adding comment is sometimes a little time-consuming. But I think it is necessary. I recommend adding comments as much as you can if you feel needed.
If you don't have many times, you can consider separating code into sections with MARKers like the comments above, that would be enough.